conf/logback.xml

```diff
@@ -34,6 +34,8 @@
   <logger name="controllers.StatsController" level="WARN"/>
 
   <logger name="nearline-entries-for-status-stream" level="DEBUG"/>
+  <logger name="models.UnclogOutputIndexer" level="DEBUG"/>
```

Comment:
can bump these up to INFO for deployment

Reply:
Done.